frontend-api/src/test/java/uk/gov/di/authentication/frontendapi/lambda/AuthenticationAuthCodeHandlerTest.java

```diff
@@ -160,17 +183,15 @@ void shouldReturn200AndSaveNewAuthCodeRequest() throws Json.JsonException {
                         anyString(),
                         eq(false));
         assertThat(result, hasStatus(200));
-        var authorizationResponse = new AuthCodeResponse(TEST_AUTHORIZATION_CODE, TEST_STATE);
-        assertThat(result, hasBody(objectMapper.writeValueAsString(authorizationResponse)));
+        assertTrue(new JSONObject(result.getBody()).has("location"));
```

Comment:
Here, is it worth explicitly asserting on the full body of the response. So asserting that it is { "location": "some-redirect-uri?with=params" }? It's nice as a bit of documentation as to what's returned by the endpoint, and eliminates the possibility that e.g. location is null